panels/user-accounts/um-app-permissions.c

```diff
@@ -250,10 +251,19 @@ update_categories_from_language (UmAppPermissions *self)
   /* Fill in the age menu */
   g_menu_remove_all (self->age_menu);
 
+    {
+      g_autofree gchar *action = g_strdup_printf ("permissions.set-age(uint32 %u)", (guint32) -1);
+      g_menu_append (self->age_menu, _("Disabled"), action);
+    }
+
   for (i = 0; entries[i] != NULL; i++)
     {
       g_autofree gchar *action = g_strdup_printf ("permissions.set-age(uint32 %u)", ages[i]);
 
+      /* Prevent the unlikely case that one of the real ages is the same as our
+       * special ‘disabled’ value. */
+      g_assert (ages[i] != (guint32) -1);
```

Comment:
I'd rather move all these `(guint32) -1` to a `#define AGE_DISABLED ((guint32) -1)` (I'm sure there would be a better name for that though!). To make this and the lines below a bit more semantic.